.github/workflows/publish-open-vsx.yml

```diff
@@ -0,0 +1,45 @@
+# Publish the extension to the open-vsx registry at https://open-vsx.org
```

Comment:
Can this separate workflow file now be deleted, since you have it as part of the `Release` workflow?